open-metadata-implementation/governance-servers/security-sync-services/security-sync-services-server/src/main/java/org/odpi/openmetadata/securitysyncservices/processor/SecuritySyncEventProcessor.java

```diff
@@ -168,7 +168,7 @@ private GovernedAssetListAPIResponse getGovernedAssets() {
         try {
 
             ResponseEntity<String> result = restTemplate.exchange(governanceEngineURL, HttpMethod.GET, entity, String.class);
-            return (GovernedAssetListAPIResponse) mapToObject(result, GovernedAssetListAPIResponse.class);
+            return (GovernedAssetListResponse) mapToObject(result, GovernedAssetListResponse.class);
         } catch (HttpStatusCodeException exception) {
             log.debug("Unable to get the governed assets!");
```

Comment:
same here, exception should be logged or thrown
debug message might  not be shown and is not very helpful without more info about request